src/routes/blog/post/cors-error/+page.markdoc

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: Solving CORS errors with appwrite
+description: Understanding why you are getting CORS error when sending request to an appwrite backend, and how to debug
+date: 2024-01-16
+cover: /images/blog/cors-error/cors_cover.png
+timeToRead: 3
+author: dennis-ivy
+category: engineering
+featured: false
+---
+
+I want to address an issue I've seen popping up on stack overflow and the appwrite discord server and address some of the reasons you may be getting this error, and walk you through some of the steps you can take to try and resolve it as well.
```

Comment:
Capitalize `Stack Overflow` `Appwrite` and `Discord`

Reply:
@divanov11 Take a look here at Laura's comments.